README.md

```diff
@@ -44,6 +44,14 @@ An identifier that can be used to deduplicate deployments - Default: `${GITHUB_R
 
 Send deploy using Docker container if "true" - use OpsLevel CLI otherwise - Default: `true`
 
+## Reporting Deploy with Docker or OpsLevel CLI
+
+This action can be run using either Docker or the OpsLevel CLI, depending on the value of the `use_docker` input.
+
+While both behave identically, this action will run using Docker by default - see [./with-docker](./with-docker).
```

Comment:
Should we call out that this is basically a Github restriction if you don't run "linux-based" github actions runners.  This can be on their cloud runners or in self-hosted runners that you run on windows or mac.  🤔